tests/conditional_compilation/proc_utils.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+""" Common utilities for working with processes.
+"""
+
+import logging
+import subprocess
+
+
+def cmd_exec(args, log=None, verbose=True):
```

Comment:
Perhaps it would be better to reuse such from https://github.com/openvinotoolkit/openvino/blob/master/tests/time_tests/scripts/run_timetest.py#L23 and move to some global utilities?

Reply:
Moved to tests library. Lets have a separate commit for other tests refactoring - there is bigger tech debt there